scripts/add_extra_components.py

```diff
@@ -187,14 +187,20 @@ def attach_stores(n, costs, config):
     if ("csp" in config["renewable"].keys()) and (
```

Comment:
Hi @davide-f, I found another bug here. 

The first check should check csp is present in `extendable_carriers:Generator` not `renewables` as `renewables` only check if csp is present in atlite.

Reply:
Integrated yesterday, sorry for not saying that. I believe we can resolve this.
Many thanks!